packages/bpk-foundations-android/tokens/base.android.xml

```diff
@@ -176,16 +181,26 @@
   <color name="TEXT_QUATERNARY_LIGHT_COLOR" category="text-colors" originalTag="SKY_GRAY_TINT_03">#ff8f90a0</color>
   <color name="TEXT_SECONDARY_DARK_COLOR" category="text-colors" originalTag="SKY_GRAY_TINT_04">#ffb2b2bf</color>
   <color name="TEXT_SECONDARY_LIGHT_COLOR" category="text-colors" originalTag="SKY_GRAY_TINT_02">#ff68697f</color>
+  <property name="TEXT_SM_EMPHASIZED_FONT_SIZE" category="font-sizes" originalTag="FONT_SIZE_SM">14sp</property>
```

Comment:
to match the hero styles should we name these `TEXT_SM-EMPHASIZED_FONT_SIZE` etc to make the parsing easier? (- instead of _ between sm & emphasized)